mlrun/projects/project.py

```diff
@@ -3035,7 +3035,6 @@ def build_image(
         overwrite_build_params: bool = False,
         requirements_file: str = None,
         extra_args: str = None,
-        force_build: bool = False,
```

Comment:
Better leave for bc but not using it

Reply:
+1 to that - keep it, mark deprecated, and warn that it's ignored and always `True`.

Reply:
it was introduced in 1.6.0 so it's safe to delete ;)